package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@tanstack/react-query": "^5.54.1",
     "appwrite": "^15.0.0",
+    "dallas-software-developers": "file:",
```

Comment:
I'm not sure what this is for. Can you remove it?

Reply:
Updated the PR!